.ci_support/win_64_numpy2.0python3.9.____cpython.yaml

```diff
@@ -1,13 +1,15 @@
 c_compiler:
 - vs2019
+c_stdlib:
+- vs
 channel_sources:
-- conda-forge
+- conda-forge/label/numpy_rc,conda-forge
```

Comment:
Think this is leftover from the `rc` branch. Though not seeing logic here that would cause it to be added. Will try re-rendering

Reply:
Don't you have to have that to get numpy 2 for building?

Reply:
Ah you are right 🤦‍♂️

Sorry misread this as `numba_rc` instead of `numpy_rc`

Ignore me 😅